src/main/java/sirius/biz/protocol/JournalEntry.java

```diff
@@ -67,11 +69,16 @@
     private String targetId;
 
     /**
-     * Contains the {@code toString()} of the entity which was changed.
+     * Contains the identifier of the changed entity.
+     * <p>
+     * The identifier is composed of {@link EntityDescriptor#getType()}-{@link BaseEntity#getIdAsString()}
+     * For entities using {@link JournalData}, it is equal to the {@link #TARGET_TYPE}-{@link #TARGET_ID}
+     * For entities using {@link DelegateJournalData}, it will contain the type and if of the entity which
+     * caused the journal entry, since the target fields contains the parent entity owning this record.
      */
-    public static final Mapping TARGET_NAME = Mapping.named("targetName");
+    public static final Mapping CONTENT_IDENTIFIER = Mapping.named("contentIdentifier");
     @SearchContent
-    private String targetName;
+    private String contentIdentifier;
```

Comment:
This is a breaking change, isnt it?

Reply:
No, the field wasn't used by no one unless a template. (Technically speaking yes, but without effect to us).

Reply:
Technically we dropped the unused field targetName and introduced the new field contentIdentifier :-P